lib/sgtn-client/util/locale-util.rb

```diff
@@ -10,6 +12,11 @@ module SgtnClient
                   }
 
     class LocaleUtil
+        def self.get_best_locale(locale)
+            flocale = fallback(process_locale(locale))
+            flocale = SgtnClient::Config.configurations.default if flocale == get_source_locale()
+            return flocale
```

Comment:
Change 'en' to 'default' will cause a request to server with locale 'default'.
```suggestion
            fallback(process_locale(locale))
```

Reply:
yes, this will cause it to use the source bundle

Reply:
Check server log, received a strange request.
```
2022-03-09T02:04:13.400Z        INFO    Start a request {“traceId”: 1120116155, “method”: “GET”, “ip”: “127.0.0.1”, “path”: “/i18n/api/v2/translation/products/test/versions/4.8.1/locales/default/components/i18n”, “query”: “checkTranslationStatus=false&machineTranslation=false&pseudo=false”}
2022-03-09T02:04:13.407Z        DEBUG   Get a bundle    {“traceId”: 1120116155, “name”: “test”, “version”: “4.8.1", “locale”: “default”, “component”: “i18n”}
2022-03-09T02:04:13.408Z        DEBUG   Read a bundle from storage      {“traceId”: 1120116155, “name”: “test”, “version”: “4.8.1”, “locale”: “default”, “component”: “i18n”}
2022-03-09T02:04:13.413Z        ERROR   Fail to get translation for test/4.8.1/default/i18n: open tests/testdata/bundles/test/4.8.1/i18n/messages_default.json: no such file or directory       {“traceId”: 1120116155}
```